src/utils/graphqlConversions.ts

```diff
@@ -0,0 +1,138 @@
+import { GetInmateQuery } from "src/graphql/GetInmate.generated";
+import { Call, MeetingStatus, Visit } from "src/typings/Call";
+import { MeetingStatus as GraphqlMeetingStatus } from "src/api/graphql";
+import { MeetingType as GraphqlMeetingType } from "src/api/graphql";
+import { MeetingType } from "src/typings/Common";
+import { GetVisitorQuery } from "src/graphql/GetVisitor.generated";
+
+type GraphqlMeeting =
+  | GetInmateQuery["inmate"]["meetings"]["edges"][0]["node"]
+  | GetVisitorQuery["visitor"]["meetings"]["edges"][0]["node"];
+
+const graphQlMeetingStatusToMeetingStatus = (status: GraphqlMeetingStatus) => {
+  const statuses = {
```

Comment:
to ensure completeness:

```suggestion
  const statuses: Record<GraphqlMeetingStatus, MeetingStatus> = {
```